src/components/SimpleTable.tsx

```diff
@@ -129,13 +109,11 @@ const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
               <Flex align="center">{type}</Flex>
             </Td>
             {url && (
-              <Td color="primary">
-                <Flex align="center" justify="flex-end">
-                  ↗
-                </Flex>
+              <Td textAlign="end">
+                <ButtonLink to={url}>Go to {name}</ButtonLink>
```

Comment:
Not sure about this but the rows look too tall with the current size of the button. Could we try to use a smaller version of the button instead? to make the entire table more compact.

cc @nloureiro 
```suggestion
                <ButtonLink to={url} size="sm">Go to {name}</ButtonLink>
```